pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java

```diff
@@ -2322,6 +2382,7 @@ public CmdNamespaces(Supplier<PulsarAdmin> admin) {
         jcommander.addCommand("grant-permission", new GrantPermissions());
         jcommander.addCommand("revoke-permission", new RevokePermissions());
 
+        jcommander.addCommand("subscription-permission", new SubscriptionPermissions());
```

Comment:
get-subscription-permission ?

Reply:
To be consistent with here(L2358)
https://github.com/apache/pulsar/blob/04ddc9b0324e6d09b6926bbfe5402f006ad633c3/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java#L2358-L2360